orchestra/conducting.py

```diff
@@ -149,6 +154,7 @@ def flow(self):
             if self.get_workflow_state() not in states.ABENDED_STATES:
                 # Set the initial workflow context.
                 init_ctx = dict(list(rendered_inputs.items()) + list(rendered_vars.items()))
+                init_ctx = dx.merge_dicts(init_ctx, self.get_workflow_parent_context(), True)
```

Comment:
@bigmstone This is the fix that made your patch works.  There's no need to put the parent context under `__vars`.  Putting the context dictionary under `__vars` is handle by the YAQL and Jinja evaluators.  The other changes I made are just cosmetic, tests, & cleanups.